app/controllers/users_controller.rb

```diff
@@ -0,0 +1,33 @@
+class UsersController < ApplicationController
+  def show
+    @user = User.find_by id: params[:id]
+  end
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new(user_params)
+    if @user.save
+      flash[:success] = "Welcome to Sample App!"
+      redirect_to @user
+    else
+      render :new
+    end
+  end
+
+  private
+
+  def user_params
+    params.require(:user).permit :name, :email, :password, :password_confirmation
+  end
+
+  def load_user
```

Comment:
you defined this function but not call anywhere

you can call it by before action:
```
  before_action :load_user, only: :show 
```
and remove line 3
```
   @user = User.find_by id: params[:id]
```